src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -157,6 +157,15 @@ private void buildPaymentTab(ConfigBuilder<CaseData, State, UserRole> configBuil
             .label("LabelPaperForm-App1PaymentHeading", PAPER_FORM_PAYMENT_OTHER_DETAILS, "### Paper Form Payment Details")
             .field("paperFormSoleOrApplicant1PaymentOtherDetail", PAPER_FORM_APPLICANT_1_PAYMENT_OTHER_DETAILS)
             .field("paperFormApplicant2PaymentOtherDetail", PAPER_FORM_APPLICANT_2_PAYMENT_OTHER_DETAILS)
+            .label("Applicant2Solicitor-PaymentHeading", "applicant2SolAppliedForFinalOrder=\"Yes\"", "### Respondent Solicitor")
```

Comment:
Introduce constant for this string? `"applicant2SolAppliedForFinalOrder=\"Yes\""`

Reply:
Good idea - done